decentraland/unity-explorer
# Fix: genesis scene (137,-3) not always triggering emotes

## What does this PR change?
Fix #1556

Issue was arising only for the green floor on the scene. After some tests it seems that there is a hardcoded y-position `if`  check in the js-code. So, to make it work, I adjusted the CharacterController collider by 5 cm (from 0.9 to 0.85)


## How to test the changes?

1. Launch the explorer
2. Check mentioned bug-ticket for the repro-steps
3. Also check that "look and feel" of the player locomotion remained same as previously

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
